docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+# Log examples
+
+To help you understand how garbage collection (GC) processes memory for your application and how these processes are recorded, a number of annotated log examples are provided from different GC policies. Each example covers a particular type of cycle from a particular policy. You will learn how to interpret the XML elements in the example log and determine the characteristics of the cycle that has been recorded.
+
+## `gencon` examples
+
+As detailed in [`gencon` policy (default)](gc.md#gencon-policy-default), this policy uses two types of cycle; a partial GC cycle and a global GC cycle. By default, the partial GC cycle runs a *stop-the-world* (STW) *scavenge* operation but can run a concurrent *scavenge* operation ([-Xgc:concurrentScavenge](xgc.md#concurrentscavenge)) if enabled at run time.
+
+The start of a `gencon` cycle is recorded in the log by the following elements and attributes:
+
+| GC cycle | Value of `type` attribute of the `<cycle-start>` and `<cycle-end` ``elements | Element that logs the cycle trigger|  Trigger reason|
+|----------|----------------------------|------------------------------|------------------|
+|Global    | `global`                   | `<concurrent-kickoff>`       | Low free memory tenure area threshold reached. Cycle trigger element is located before the `<cycle-start>` element|
+|Partial | `scavenge`                   | `<af-start>`                 |Allocation failure. Cycle trigger element is located before the `<cycle-start>` element|
+
+You can use the `type` attribute of the `<gc-start>` and `<gc-end>` elements to locate a particular cycle. YOu can also locate a particular type of cycle by searching for the element that records the cycle trigger, which is located before the `<cycle-start>` element.
+
+You can analyze the increments and operations that are associated with a particular type of cycle by locating and interpreting the elements in the following table:
+
+
+| GC process             | Elements that log the start and end of the event  | Details |
+|------------------------|------------------------------|------------------------------|
+|GC cycle                |`<cycle-start>`, `<cycle-end>`| The start and end of a GC cycle.|
+|GC STW increment        |`<gc-start>`, `<gc-end>`      | The start and end of a GC increment that begins with a pause.
+|GC STW increment        | `<concurrent-kickoff>`       | The start of the initial GC increment of the global concurrent cycle that begins the initial mark operation
+|GC STW increment        | `<concurrent-global-final> ` | The start of the final GC increment of the global concurrent cycle that executes the final collection|
+|GC operations and sub-operations| `<gc-op>`                    | A GC operation such as mark or sweep, or a suboperation such as class unload. |
```

Comment:
I would say that class unloading is a _sub_ operation. It's somewhat equivalent to sweep, but while sweep operates on object/heap, class unloading operates on class (which for the most part are outside of object heap).
 If something is sub operation it's card cleaning or remember set processing (parts of of global marking)

Reply:
@amicic Hi Aleksandar, can you please clarify? - that class unload is referred to as a sub operation in the table.